packages/protocol/lib/test-utils.ts

```diff
@@ -439,30 +439,28 @@ export async function assumeOwnership(contractsToOwn: string[], to: string, prop
 		artifacts
 	)
 	const registry: RegistryInstance = await getDeployedProxiedContract('Registry', artifacts)
+  // Enough to pass the governance proposal unilaterally (and then some).
+  const tenMillionCELO = '10000000000000000000000000'
```

Comment:
The indention looks wacky in the Github Diff

Reply:
Hmm looks good in my editor, and prettify has no objections. If you check out this branch do you see a misalignment?

Reply:
<img width="919" alt="Screen Shot 2021-04-15 at 2 13 44 PM" src="https://user-images.githubusercontent.com/571810/114939084-2cde8780-9dfd-11eb-9d8e-4ac8ccbc0015.png">
 Weird, but yeah checking out does seem to be different